Privacy Policy.md

```diff
@@ -30,19 +35,31 @@ We will NEVER sell your data to any third-parties, although we use certain servi
+If you wish to request deletion of any information stored on our servers, including any requests under GDPR or DMCA, please use the account deletion / reporting in-app if applicable or otherwise contact us at [urgent@revolt.chat](mailto:urgent@revolt.chat).
 
-*Exceptions to third-parties service providers which is used to process your data as mentioned above.*
+_Exceptions to third-parties service providers which is used to process your data as mentioned above._
 
 ### Our commitment to children's privacy
 
-We follow [The Children’s Online Privacy Protection Act (“COPPA”)](https://www.ftc.gov/enforcement/rules/rulemaking-regulatory-reform-proceedings/childrens-online-privacy-protection-rule), which requires online service providers to obtain parental consent before they knowingly collect personally identifiable information online from children who are under 13 years of age. This applies to every user of our platform, regardless of different consent laws in different countries. We do not knowingly collect or solicit personally identifiable information from children under 13 years of age. If we learn we have collected personal information from a child under 13 years of age, we will immediately delete that information from our servers. If you are a parent or guardian of a child under the age of 13 and believe that they have disclosed personal information to us, please contact us at [contact@revolt.chat](mailto:contact@revolt.chat).
+We abide by [the children's consent guidelines as laid out by the GDPR](https://gdpr-info.eu/art-8-gdpr/) and [The Children’s Online Privacy Protection Act (“COPPA”)](https://www.ftc.gov/enforcement/rules/rulemaking-regulatory-reform-proceedings/childrens-online-privacy-protection-rule), which requires online service providers to obtain parental consent before they knowingly collect personally identifiable information online from children who are under 16 and 13 years of age respectively, and in the case of the GDPR, may vary depending on the EU member state.
+
+We do not knowingly collect or solicit personally identifiable information from children under the minimum legal age to use the service. If we learn we have collected personal information from a child under the minimum legal age to use the service, we will immediately delete that information from our servers. If you are a parent or guardian of a child under the age and believe that they have disclosed personal information to us, please contact us at [contact@revolt.chat](mailto:contact@revolt.chat). As a concerned user of the platform, you may also report underage users by using the in-app report feature.
+
+### Your rights under the GDPR
+
+The data controller of your personal data is Pawel Makles, an individual who operates Revolt as a sole proprietorship. You can contact him at [data-controller@revolt.chat](mailto:data-controller@revolt.chat). The data controller is responsible for deciding how and why we process your personal data.
```

Comment:
I think it would be better to just use your real name

Reply:
in the email address?

Reply:
I suggest replacing the email address to _dpo@revolt.chat (Data Protection Officer)_ for simplicity.